src/components/Tracking.tsx

```diff
@@ -12,11 +12,15 @@ import { closestPoint, toNumber } from '../utils';
 export interface ClosestInfo<T extends ClosestMethods> {
   point: SeriesPoint;
   label: string;
-  axis: T extends 'euclidean'
+  axis: T extends ClosestMethods.euclidean
     ? Record<'x' | 'y', PlotAxisContext>
     : PlotAxisContext;
 }
-export type ClosestMethods = 'x' | 'y' | 'euclidean';
+export enum ClosestMethods {
+  x = 'x',
+  y = 'y',
+  euclidean = 'euclidean',
+}
```

Comment:
For libraries, I tend to prefer using unions instead of enums because they are easier to use (no need to import, autocomplete just by writing `''`, interoperable with values that come from somewhere that doesn't know about `react-plot`)